cellprofiler_core/preferences/__init__.py

```diff
@@ -438,6 +451,11 @@ def resources_root_directory():
     SPP_PIPELINE_AND_FILE_LIST,
 ]
 
+# Registry Key Types
+bool_keys = [SHOW_SAMPLING, TELEMETRY, TELEMETRY_PROMPT, STARTUPBLURB]
+int_keys = [SKIPVERSION, OMERO_PORT, MAX_WORKERS, JVM_HEAP_MB]
+float_keys = [TITLE_FONT_SIZE, TABLE_FONT_SIZE, PIXEL_SIZE]
```

Comment:
Should the variable names be all caps since they're module constants?

Also, these could potentially be sets which would make the `key in <keys>` checks marginally faster.